core/controllers/contributor_dashboard_admin.py

```diff
@@ -529,3 +530,191 @@ def _get_complete_translation_contribution_stats(
             })
 
         return response_translation_contribution_stats_dicts
+
+
+class ContributorDashboardAdminStatsHandlerNormalizedPayloadDict(TypedDict):
+    """Dict representation of ContributorDashboardAdminStatsHandler's
+    normalized_request dictionary.
+    """
+
+    page_size: int
+    offset: int
+    language_code: str
+    sort_by: Optional[str]
+    topic_ids: Optional[List[str]]
+    num_days_since_last_activity: Optional[int]
+
+
+class ContributorDashboardAdminStatsHandler(
+    base.BaseHandler[
+        ContributorDashboardAdminStatsHandlerNormalizedPayloadDict,
+        Dict[str, str]
+    ]
+):
+    """Return Contributor Admin Dashboard Stats for supplied parameters.
+    """
+
+    GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+    URL_PATH_ARGS_SCHEMAS = {
+        'contribution_type': {
+            'schema': {
+                'type': 'basestring'
+            }
+        },
+        'contribution_subtype': {
+            'schema': {
+                'type': 'basestring'
+            }
+        }
+    }
+    HANDLER_ARGS_SCHEMAS = {
+        'POST': {
+            'page_size': {
+                'schema': {
+                    'type': 'int'
+                },
+                'default_value': 20
+            },
+            'offset': {
+                'schema': {
+                    'type': 'int'
+                }
+            },
+            'language_code': {
+                'schema': {
+                    'type': 'basestring'
+                },
+                'default_value': None
+            },
+            'sort_by': {
+                'schema': {
+                    'type': 'basestring'
+                },
+                'default_value': None,
+                'choices': constants.CD_ADMIN_STATS_SORT_OPTIONS
+            },
+            'topic_ids': {
+                'schema': {
+                    'type': 'list',
+                    'items': {
+                        'type': 'basestring'
+                    }
+                },
+                'default_value': None
+            },
+            'num_days_since_last_activity': {
+                'schema': {
+                    'type': 'int'
+                },
+                'default_value': None
+            }
+        }
+    }
+
+    @acl_decorators.can_access_contributor_dashboard_admin_page
+    def post(
```

Comment:
Actually I took the reference of [this](https://github.com/oppia/oppia/blob/develop/core/controllers/topics_and_skills_dashboard.py#L256) code used for getting data for skills dashboard. Since we are implementing pagination in the backend itself we need to send data via this API. I am not sure about this though, can you please give me suggestion whether should I change it. If yes I'll have to make the changes and also change the tests so will resolve other comments after getting this one resolved.

Reply:
Yes change it to GET as it's a request to fetch data, there are a lot of issues with using post in place of GET. Also, thanks for reporting the [issue](https://github.com/oppia/oppia/blob/develop/core/controllers/topics_and_skills_dashboard.py#L256) in develop., I'll report it to the LaCE team and get it fixed.

Reply:
@kshitij01042002 Is this resolved?